src/api-transition-helpers.coffee

```diff
@@ -0,0 +1,17 @@
+LineNumberGutterName = 'line-numbers'
+
+# Given decorationParams/Properties from the 1.0 API, applies changes to enable
+# the 2.0 API under the covers. In 2.0, the `trueType` property will become `type`.
+translateDecorationParamsOldToNew = (decorationParams) ->
```

Comment:
On second thought, the reason I wanted to do the isTrueType is to avoid having places in the Atom Core code where checking for type='line-gutter' will work. Everything should instead conform to the new API, which requires type='gutter' and the name.

When it comes time to transition, it seemed simpler to do a rename from isTrueType->isType than to change filter arguments.